jcg_testcases/src/main/resources/js/AnonymousFunctions.md

```diff
@@ -0,0 +1,65 @@
+# Anonymous Functions
+Anonymous functions, also called Lambda functions, are functions that are not bound to an identifier.
+
+## AF1
+[//]: # (MAIN: global)
+Test the use of an anonymous function as an argument to another function.
+
+```json
+{
+  "nodes": [
+    "<global>", "AF1.foo", "AF1.<anonymous:6>"
+  ],
+  "directLinks": [
+    ["<global>", "AF1.foo"],
+    ["AF1.foo", "AF1.<anonymous:6>"]
+  ],
+  "indirectLinks": []
+}
+```
+```js
+// af/AF1.js
+
+function foo(f) {
+    return f(1);
+}
+
+foo(function(x) {
+    return x+1;
+});
+```
+[//]: # (END)
+
+## AF2
+[//]: # (MAIN: global)
+Test the use of multiple anonymous functions as arguments to another function.
+
+```json
+{
+  "nodes": [
+    "<global>", "AF2.foo", "AF2.<anonymous:8>", "AF2.<anonymous:11>"
+  ],
+  "directLinks": [
+    ["<global>", "AF2.foo"],
+    ["AF2.foo", "AF2.<anonymous:8>"],
+    ["AF2.foo", "AF2.<anonymous:10>"]
+  ],
+  "indirectLinks": []
+}
+```
+```js
+// af/AF2.js
+
+function foo(f1, f2) {
+    const x = f1(1);
+    const y = f2(1);
+    return x + y;
+}
+
+foo(function(x) {
+    return x;
+}, function(x) {
```

Comment:
As above, and also don't start the second argument on the line where the first is, just to avoid any confusion on the lines where which function is defined?